ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/NewTabPageViewController.swift

```diff
@@ -1342,7 +1342,7 @@ extension NewTabPageViewController {
     }
 
     UmaHistogramRecordValueToBucket(
-      "Brave.NTP.NewTabsCreated.2",
+      "Brave.NTP.NewTabsCreated.3",
```

Comment:
I assume there are no differences in the buckets/answer?

Reply:
correct